CHANGELOG.md

```diff
@@ -24,6 +24,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### [shopsys/framework]
+#### Changed
+- [#271 - Complete refactoring of feeds functionality](https://github.com/shopsys/shopsys/pull/271)
+  - modules are responsible for querying the data to improve performance
+  - interfaces from package product-feed-interface are not used anymore as they were only important with open-box architecture
+  - only relevant data is fetched from the database, should result in enhanced performance
+  - FeedInterface and FeedInfoInterface define the way feeds are registered in the system
+  - FeedExport is responsible for the actual generation of a file in batches on a specific domain
+  - FeedRenderer is responsible for rendering the feed from Twig template
+  - FeedPathProvider is responsible for providing the correct filepath and url to the specified feed on a domain
+  - ProductUrlsBatchLoader and ProductParametersBatchLoader are responsible for loading product data in batches 
+  - cron modules use the logger for debug information
+  - DailyFeedCronModule is responsible for continuation of the correct feed after waking up
+
+### [shopsys/product-feed-google], [shopsys/product-feed-heureka], [shopsys/product-feed-heureka-delivery], [shopsys/product-feed-zbozi]
```

Comment:
I am not sure this is a valid notation in our changelog...

Reply:
Me neither. But I'm 100 % positive that I don't want to copy-paste it four times. I can do it if you insist.

Reply:
let us try to keep it as it is :laughing: